onnxruntime/core/providers/cpu/math/logsoftmax.cc

```diff
@@ -12,6 +14,9 @@ namespace onnxruntime {
 
 template <>
 Status LogSoftmax<float>::Compute(OpKernelContext* ctx) const {
+  auto ctx_internal = static_cast<OpKernelContextInternal*>(ctx);
+  auto tp = ctx_internal->GetOperatorThreadPool();
```

Comment:
Is it worth considering moving GetOperatorThreadPool to OpKernelContext? Seems like we have enough places using it to warrant it being part of the public class.

Reply:
Absolutely.